README.md

```diff
@@ -290,6 +297,7 @@ The following fields are supported:
 | `groups` | list of objects | List of statically defined groups. |
 | `groups[].name` | string | *Required.* The unique identifying name of the group. |
 | `groups[].long_name` | string | *Required.* The human-readable descriptive name of the group. |
+| `groups[].dont_manage_members` | bool | Do not manage members of this group. This is useful if only the group should be created statically. |
```

Comment:
Double negation causes confusion and should be avoided. Instead of `"dont_manage_members": true`, I'd rather want `"manage_members": false`. A default value of true can be realized by making the bool field a pointer member and interpreting `nil` as true.

Reply:
That's what I was missing